tests/explorer.tests/Setup/ExplorerTestFixture.cs

```diff
@@ -50,36 +53,51 @@ public ExplorerTestFixture()
 
         public Container Container { get; }
 
-        public static string GenerateVcrFilename(object caller, [CallerMemberName] string name = "") =>
-            Cassette.GenerateVcrFilename(caller, name);
-
-        public TestScope PrepareTestScope() => new TestScope(Container);
-
-        public QueryableTestScope SimpleQueryTestScope(
-            string dataSource,
-            string vcrFilename) =>
-            PrepareTestScope()
-                .LoadCassette(vcrFilename)
-                .WithConnectionParams(ApiUri, dataSource);
-
-#pragma warning disable CA2000 // Call System.IDisposable.Dispose on object (Allow calling context to dispose the scope.)
-        public ComponentTestScope SimpleComponentTestScope(
+        public async Task<TestScope> CreateTestScope(
             string dataSource,
             string table,
             string column,
-            DColumnInfo columnInfo,
-            string vcrFilename) =>
-            PrepareTestScope()
-                .LoadCassette(vcrFilename)
-                .WithConnectionParams(ApiUri, dataSource)
-                .WithContext(dataSource, table, column, columnInfo);
-#pragma warning restore CA2000 // Call System.IDisposable.Dispose on object
+            object caller,
+            RecordingOptions recordingOptions = RecordingOptions.SuccessOnly,
+            VCRMode vcrMode = VCRMode.Cache,
+            [CallerMemberName] string callerMemberName = "")
+        {
+            var vcrFileName = Cassette.GenerateVcrFilename(caller, callerMemberName);
+            var columnInfo = await GetColumnInfo(dataSource, table, column);
+            return new TestScope(Container, ApiUri, dataSource, table, column, columnInfo, vcrFileName, vcrMode, recordingOptions);
+        }
 
         public void Dispose()
         {
             Container.Dispose();
         }
 
+        private async Task<DColumnInfo> GetColumnInfo(string dataSource, string table, string column)
+        {
+            var apiClient = Container.GetNestedContainer().GetInstance<JsonApiClient>();
```

Comment:
The call to `GetNestedContainer()` is redundant here. Also I think it would create a new container instance that is not automatically disposed. 

```suggestion
            var apiClient = Container.GetInstance<JsonApiClient>();
```